client/components/GamePage.js

```diff
@@ -89,8 +89,21 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
+const BEFORE_UNLOAD_MESSAGE = 'Exit drawing game?';
+
 export default () => {
   const classes = useStyles();
+
+  const handleBeforeUnload = (e) => {
```

Comment:
this means it'll prompt you when you try to navigate away no matter what the game state is, right? I guess ideally we would only set this once a game starts? Still better this way than not having it at all though

Reply:
Yeah good point, updated both flows to only kick in once a game has started :-)